### PR TITLE
Rephrased sentences to match new reality

### LINKED ISSUE #1031: [doc] Issue in "SSL for RGW in 7.1" - got an unexpected keyword argument 'rgw_frontend_ssl_key'
### Type of report
   - [ x] bug
   - [ ] feature request

### Source Document[
https://documentation.suse.com/ses/7.1/single-html/ses-deployment/#cephadm-deploy-using-secure-ssl-access

### Summary
the rgw_frontend_ssl_key does NOT exist in 7.1 / pacific anymore!

### Steps To Reproduce
Follow the docs and apply rgw.yaml - you will get the above error

### Expected Results
Deployment works without a problem

### Actual Results
Fails

### Notes
Seems the admin and deployment guides need to cover the changed for pacific!

### PR DESCRIPTION
**Pull request by tbazant**
fixes #1031

### CODE REVIEW

File: xml/deploy_core.xml [hunk 1]
```diff
@@ -548,10 +548,10 @@ spec:
...
     <title>Using secure SSL access</title>
     <para>
-     To use a secure SSL connection to the &ogw;, you need a pair of valid SSL
-     certificate and key files (see <xref linkend="ceph-rgw-https"/> for more
-     details). You need to enable SSL, specify a port number for SSL
-     connections, and the SSL certificate and key files.
+     To use a secure SSL connection to the &ogw;, you need a file containing
+     both valid SSL certificate and key (see <xref linkend="ceph-rgw-https"/>
+     for more details). You need to enable SSL, specify a port number for SSL
+     connections, and the SSL certificate and key file.
```
Comment (trociny) [inline xml/deploy_core.xml:554]: nit: I would say just: "and the SSL certificate file".

File: xml/deploy_core.xml [hunk 2]
```diff
@@ -587,16 +586,13 @@ spec:
 </screen>
     <tip>
      <para>
-      Instead of pasting the content of SSL certificate and key files, you can
-      omit the <literal>rgw_frontend_ssl_certificate:</literal> and
-      <literal>rgw_frontend_ssl_key:</literal> keywords and upload them to the
-      configuration database:
+      Instead of pasting the content of a SSL certificate and key file, you can
```
Comment (trociny) [inline xml/deploy_core.xml:589]: the same here: just "a SSL certificate file".

**Review by trociny — APPROVED** [review]
otherwise LGTM

**Review by dariavladykina — COMMENTED** [review]
LG, just a small correction

**Pull request merged**